ui/public/config.json

```diff
@@ -59,5 +67,119 @@
   "basicZoneEnabled": true,
   "multipleServer": false,
   "allowSettingTheme": true,
-  "docHelpMappings": {}
+  "docHelpMappings": {
```

Comment:
@RodrigoDLopez doc help mappings are auto-generated while building UI and shouldn't be a part of this PR

Reply:
@shwstppr can you explain how to deal with having such an extended list of documentation links, please? (I am not saying that these are appropriate or not and should stay in or not, just curious)

Reply:
@DaanHoogland This list is auto-generated when building UI https://github.com/apache/cloudstack/blob/main/ui/prebuild.sh#L19-L47.
It is cleared and set to empty once `npm run build` finishes. At times when the build is aborted or fails the list doesn't get cleared. It must've been added by author by mistake.

Reply:
oh. i didn't see that.
thanks for the review.